### ISSUE TITLE
"Auto saved" notification shown with every keystroke

### ISSUE THREAD

**Issue by MarcelvC** [w8tcha/CKEditor-AutoSave-Plugin#69]
A new "auto saved" notification is shown with every keystroke. In addition, interaction with other editor buttons is impeded or prevented, even when _messageType_ is set to "no" in the config file.

The issue only occurred after updating to the latest version of ckeditor (4.6.1). 

![auto_save_bug](https://cloud.githubusercontent.com/assets/12641226/21544160/11a844e2-cdcd-11e6-8190-884b373ae2f1.png)

**Comment by w8tcha** [comment]
Are you using the latest plugin,js frrom the source repository?

**Comment by MarcelvC** [comment]
No, I downloaded it from http://ckeditor.com/addon/autosave. It was only a few days ago though so I think it was version 0.16.

**Comment by briceparent** [comment]
The version number doesn't change with every commit. When you download from this page, you get a v0.16, as in the repo, but the source is not up-to-date.
I've had the same problem with an old issue (an undeclared `event` variable). It's not easy to know if your source is up-to-date, the easiest way is to redownload it from Github.

**Comment by w8tcha** [comment]
Yes correct the repository is more new then the release on ckeditor.com i no release a new version after each commit

**Comment by briceparent** [comment]
It would be more clear for your users to have a change in the version for each commit, even just like v0.16.1 if you don't want to increase the first sub-version number.

**Comment by w8tcha** [comment]
If that helps i can do that in the future.

**Comment by briceparent** [comment]
For those who know the number doesn't follow the commits, it's not a problem anymore, but for new users, it might help.
Good job anyway, your plugin is really helping.

**Comment by MarcelvC** [comment]
Thanks a lot for your help!

Unfortunately though, even with the latest set of files from the repository the problem persists.

**Comment by w8tcha** [comment]
Are you using the default autosave config?

**Comment by MarcelvC** [comment]
I finally figured out what the problem was: I originally thought that I could just delete the old files of the plugin and replace them with the new ones. Since I used the builder to create my version of the editor that didn't work though. I had to build a new version altogether _without_ the autosave plugin but _with_ the notification plugin added. After that I could add the autosave plugin files from the repository and now everything seems to work. :-)
